framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -461,6 +461,8 @@ bool cfg_get_unfriendly_device_max_timelife_delay_days(
 
 bool bpl_cfg_get_wpa_supplicant_ctrl_path(const std::string &iface, std::string &wpa_ctrl_path)
 {
+    std::string path = "/var/run/wpa_supplicant/";
```

Comment:
```suggestion
    const char* path { "/var/run/wpa_supplicant/" };
```
Only assign string literals to const variables
Don't use std::string for string constant expressions
Favor direct brace initialization whenever possible.

Reply:
fixed